docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -243,15 +245,45 @@ import cocoapods.example.*
 
 3. Re-import the project.
 
-To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
-
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`:
+   
 ```kotlin
 import cocoapods.YandexMapKit.*
 ```
-
+   
 If you use the `packageName` parameter, you can import the library using the package name `import <packageName>`:
-
+   
 ```kotlin
 import YandexMK.YMKPoint
 import YandexMK.YMKDistance
 ```
+
+### Support for Objective-C headers with @import directives
+
+> This feature is [Experimental](components-stability.md#stability-levels-explained).
+> It may be dropped or changed at any time. Use it only for evaluation purposes.
+> We'd appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
```

Comment:
I thought we only asked for feedback in what's new? 😃

Reply:
I think in your PR it was a separate paragraph, and this structure is definitely used in WN. As for warning notes, I see different wording throughout our docs
Sorry if my comment was confusing. Maybe we should discuss a unified approach to this

Reply:
Ohhh OK! I misunderstood. Thanks for clarifying 😄